src/main/resources/avro/references.avdl

```diff
@@ -81,21 +81,12 @@ record ReferenceSet {
   string id;
 
   /**
-  The IDs of the `Reference` objects that are part of this set.
-
-  This field should be left null if the set contains a very large number of
-  `Reference`s. The `Reference`s in the set may also be obtained through a
```

Comment:
Perhaps this mention of SearchReferences could be moved to the record-level comment. It's not clean, since ReferenceSet doesn't really need to know about SearchReferences, but it could potentially save some time for clients. Alternatively, this could go into the protocol doc at the top.